packages/toolkit/src/view/model/view-settings/ModelSettingsHead.tsx

```diff
@@ -0,0 +1,171 @@
+import React from "react";
+import { EntityAvatar } from "../../../components";
+import {
+  GitHubIcon,
+  Icons,
+  Tag,
+  TabMenu,
+  getModelInstanceTaskToolkit,
+  Nullable,
+} from "@instill-ai/design-system";
+import { Model } from "../../../lib";
+
+export type ModelSettingsTabs =
+  | "overview"
+  | "api"
+  | "examples"
+  | "predictions"
+  | "versions"
+  | "settings";
+
+export type HeadProps = {
+  selectedTab: ModelSettingsTabs;
+  onTabChange: React.Dispatch<React.SetStateAction<ModelSettingsTabs>>;
+  model?: Model;
+};
+
+const OWNER = {
+  avatarUrl: "",
+  name: "",
+};
```

Comment:
@orangecoloured To avoid confusing, I am very hesitant to use empty string in this codebase, most of the time I will use null to represent empty value. I will recommend this act due to it can shape a pre "Failed Early". So whenever there has bug we can trace it quicker